sdk/loadtestservice/Azure.Developer.LoadTesting/README.md

```diff
@@ -0,0 +1,135 @@
+
+
+### Authenticate the client
+
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first use [nuget][nuget] install [`azure-identity`][azure_identity_nuget]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from Azure.Identity to use.
+
+As an example, sign in via the Azure CLI `az login` command and [DefaultAzureCredential](https://learn.microsoft.com/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) will authenticate as that user.
+
+Use the returned token credential to authenticate the client.
+
+## Key concepts
+
+The following components make up the Azure Load Testing service. The Azure Load Test client library for C# allows you to interact with each of these components through the use of clients. There is a top level client  `LoadTestingClient` with two sub-clients
+
+- `LoadTestAdministrationClient`
+
+- `TestRunClient`
+
+These sub-clients are used for managing and using different components of the service. For each method in both of these sub-clients there is a corresponding Async method in the same class, with Async with same implimentation howeven enabling async functionalities, example, if there is a method, `CreateOrUpdateTest` as a part of `LoadTestAdministrationClient` then there always exists one more function `CreateOrUpdateTestAsync` in the same client class. 
+
+### Load Test Administration Client
+
+The `LoadTestAdministration` sub-clients is used to administer and configure the load tests, app components and metrics.
+
+#### Test
+
+A test specifies the test script, and configuration settings for running a load test. You can create one or more tests in an Azure Load Testing resource.
+
+#### App Component
+
+When you run a load test for an Azure-hosted application, you can monitor resource metrics for the different Azure application components (server-side metrics). While the load test runs, and after completion of the test, you can monitor and analyze the resource metrics in the Azure Load Testing dashboard.
+
+#### Metrics
+
+During a load test, Azure Load Testing collects metrics about the test execution. There are two types of metrics:
+
+1. Client-side metrics give you details reported by the test engine. These metrics include the number of virtual users, the request response time, the number of failed requests, or the number of requests per second.
+
+2. Server-side metrics are available for Azure-hosted applications and provide information about your Azure application components. Metrics can be for the number of database reads, the type of HTTP responses, or container resource consumption.
+
+### Test Run Client
+
```

Comment:
Bring this after #### Load Test Administration Client and make it a level 4 heading as well.